fpage/templates/fpage/home.html

```diff
@@ -0,0 +1,203 @@
+{% block content %}
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Albums</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+  
+    <!-- Fontawesome for bootstrap 4-->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
+      integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+  
+      <style>
+        .home{
+            background-color: #6666a3;
+          height: 50vh !important; 
+        }
+        .page-footer {
+  background-color: #101011;
+        }
+            .last-footer {
+            background-color: #6c6c72;
+        }
+            .card-color{
+            background-color: #1e1e1f;
+        }
+      </style>
+</head>
+
+
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
Include navbar template here instead of writing the whole navbar again

Reply:
"I was using the template inheritance at first but the structure of the landing page was not coming properly so i shifted to this and i dont think we would be hardly making changes in the home page so it wont create a problem"
same issue